python/sparknlp/annotator.py

```diff
@@ -1160,9 +1164,6 @@ def setTrainValidationProp(self, v):
         self._set(trainValidationProp=v)
         return self
 
-    def setIncludeValidationProp(self, v):
-        return self._set(includeValidationProp=v)
-
```

Comment:
Still not sure why is the removal of this Param?

Reply:
As above, I can bring these in.